Assets/Scripts/Gamestate/AIManager.cs

```diff
@@ -249,7 +249,8 @@ private void FindPlayers()
         agent.stoppingDistance = ShootingTarget ? shootingStoppingDistance : itemStoppingDistance;
         try
         {
-            agent.SetDestination(DestinationTarget.position);
+            if (agent.enabled)
+                agent.SetDestination(DestinationTarget.position);
```

Comment:
I think you need to add an 
`
else
{
   aimovement.enabled = true;
}
` to preserve the behavior here